Source/WebKit/WebProcess/Extensions/API/Cocoa/WebExtensionAPIScriptingCocoa.mm

```diff
@@ -147,7 +149,12 @@ static inline void parseCSSInjectionOptions(NSDictionary *cssInfo, WebExtensionS
     parseTargetInjectionOptions(script[targetKey], parameters, outExceptionString);
     parseScriptInjectionOptions(script, parameters);
 
-    // FIXME: <https://webkit.org/b/259954> Handle script injections in the UIProcess.
+    WebProcess::singleton().sendWithAsyncReply(Messages::WebExtensionContext::ScriptingExecuteScript(WTFMove(parameters)), [protectedThis = Ref { *this }, callback = WTFMove(callback), this](std::optional<Vector<WebKit::WebExtensionScriptInjectionResultParameters>> results, WebExtensionDynamicScripts::Error error) {
```

Comment:
Is `, this` needed? I think it will complain since it isn't used from what I can tell.

Reply:
i needed for `toWebAPI`. i got a `'this' cannot be implicitly captured in this context` without it